packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -0,0 +1,315 @@
+<template>
+    <component
+        :is="url ? 'a' : 'div'"
+        :href="url"
+        :data-test-id="testId"
+        :class="[
+            $style['c-stampCard1']
+        ]"
+        @click="onClickContentCard"
+    >
+        <div
+            :class="[
+                $style['c-stampCard1-headerDetails']
+            ]"
+        >
+            <img
+                :class="[
+                    $style['c-stampCard1-icon']
+                ]"
+                :data-test-id="testIdForSection('image')"
+                :src="image"
+                :alt="title"
+            >
```

Comment:
Could you please move `>` to the last passed attribute here and in other places? It will inline it with other templates style. The same goes for first attribute to be on the same line as the element like 
```
<img :class="smth"
      :src="smth"
      :alt="smth">
```